packages/syft/src/syft/core/node/common/action/smpc_action_seq_batch_message.py

```diff
@@ -0,0 +1,131 @@
+# future
```

Comment:
Same functionality as `SMPCActionMessage` only that it keeps a list of `SMPCActionMessages`